src/util/save.ts

```diff
@@ -585,4 +585,7 @@ export const UPGRADE_COLLECTION: { [version: number]: (param: string) => string
     43: param =>
         // Bump save version to support Chongqing Rail Transit stations.
         JSON.stringify({ ...JSON.parse(param), version: 44 }),
+    44: param =>
+        // Bump save version to support Chongqing Rail Transit 2021 stations and facilities.
+        JSON.stringify({ ...JSON.parse(param), version: 45 }),
```

Comment:
`isLoop` is added in chongqingrt-basic which is already used in the production. Set this key to a default value for all this type of stations.

https://github.com/railmapgen/rmp/blob/325f55b45dd5b17f7d9ad5845562293e0ff129ab/src/util/save.ts#L392